serde::export::From<std::io::Error> not implemented

When building with nightly rust on Windows, I get the following error:

```
> rustc --version
rustc 1.22.0-nightly (4502e2aa9 2017-10-03)
```

```
> cargo build
error[E0277]: the trait bound `errors::Error: serde::export::From<std::io::Error>` is not satisfied
  --> world\src\mapfile.rs:39:5
   |
39 |     write!(output, "{}", MapSave { map, legend })?;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `serde::export::From<std::io::Error>` is not implemented for `errors::Error`
   |
   = help: the following implementations were found:
             <errors::Error as serde::export::From<ron::ser::Error>>
             <errors::Error as serde::export::From<ron::de::Error>>
             <errors::Error as serde::export::From<errors::ErrorKind>>
             <errors::Error as serde::export::From<&'a str>>
             <errors::Error as serde::export::From<std::string::String>>
   = note: required by `serde::export::From::from`
```

Is this an easy fix? I'm still learning `error_chain` and Rust in general.

--- a/world/src/lib.rs
+++ b/world/src/lib.rs
@@ -110,7 +110,7 @@ pub fn roll<R: rand::Rng>(rng: &mut R) -> f32 {
 pub mod errors {
     error_chain! {
         foreign_links {
-            Io(::std::io::Error) #[cfg(unix)];
+            Io(::std::io::Error);
             RonSerialize(::ron::ser::Error);
             RonDeserialize(::ron::de::Error);
         }

